homeassistant/components/qbus/const.py

```diff
@@ -0,0 +1,14 @@
+"""Constants for the Qbus integration."""
+
+from typing import Final
+
+from homeassistant.const import Platform
+
+DOMAIN: Final = "qbus"
+PLATFORMS: list = [Platform.SWITCH]
+
+CONF_ID: Final = "id"
+CONF_SERIAL: Final = "serial"
```

Comment:
Can we rename this to SERIAL_NUMBER? SERIAL mislead me on the coordinator to think this was about serial connections.

Reply:
Done